packages/generator/src/action-function-import/parameters-interface.spec.ts

```diff
@@ -1,10 +1,13 @@
 import { unixEOL } from '@sap-cloud-sdk/util';
 import { StructureKind } from 'ts-morph';
-import { orderBreakfast } from '../../test/test-util/data-model';
+import {
+  orderBreakfast,
+  orderBreakfastNullable
+} from '../../test/test-util/data-model';
 import { parametersInterface } from './parameters-interface';
 
 describe('parameters-interface', () => {
-  it('functionImportParametersInterface', () => {
+  it('functionImportParametersInterface not nullable', () => {
```

Comment:
Consider giving a bit more context in the description of this and the test below.
I doubt that what is meant, is "the function `functionImportParametersInterface` is not nullable", but that is what I read.

Reply:
done